README.md

```diff
@@ -2,72 +2,86 @@
 
 # Lesson 6: Rails Basics
 
-We are going to create a Rails application from scratch, explaining all of the parts.  To begin, fork this repository.  Then clone your fork.  As usual, be sure not to clone into an existing git repository.  Then create a new branch called rails_basics.  This is where you will do your work. Then do:
-```
+We are going to create a Rails application from scratch, explaining all of the parts.  To begin, fork this repository.  Then clone your fork.  As usual, be sure not to clone into an existing Git repository.  Then create a new branch called `rails_basics`.  This is where you will do your work. Then do:
+
+```shell
```

Comment:
GitHub supports syntax highlighting if you tell it what language to use. For the shell commands on this page, it's not that helpful because the Rails commands don't have anything fancy, but it should help with Ruby and ERB.